apps/khix/src/app/credits/page.module.css

```
@@ -0,0 +1,749 @@
+
+.scrollCue:hover,
+.scrollCue:focus-visible {
+  color: #ffffff;
+  outline: none;
+}
+
+.scrollLine {
+  display: block;
+  width: 3.5rem;
+  height: 1px;
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)